src/linear_samplers/block_row_rand_replace.jl

```diff
@@ -0,0 +1,47 @@
+# This code was Written by Nathaniel Pritchard
+"""
+    LinSysBlkRowReplace <: LinSysBlkRowSampler 
+
+A mutable structure with fields to handle randomly subset block sampling with replacement, also known as
+vanilla block randomized kaczmarz. 
```

Comment:
- Capitalize Kaczmarz
- Reference

Reply:
I have done this and reference the Ion Necorra paper on faster randomized Kaczmarz